smartmeter/display.cpp

```diff
@@ -123,6 +127,23 @@ void Display::displayLogo()
     display->display();
 }
 
+void Display::run()
+{
+    static uint counter = 0;
+    static uint screenId = 0;
+    static uint lastScreenId = -1;
```

Comment:
uint = -1?

Reply:
This will be overwritten in the first integration.